src/components/auth/Login.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { Formik } from 'formik';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Container,
+  TextField,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+import './styles.css';
+
+const useStyles = makeStyles({
```

Comment:
Since you're using a css file here wouldn't be better to have everything together in the same CSS file?